crates/benchmark-runner/src/lib.rs

```diff
@@ -61,82 +61,67 @@ fn process_corpus<V>(
 where
     V: zkVM + Sync,
 {
-    // Take the last element, because benchmarks are setup in such a way that
-    // We only want to benchmark the last block.
-    let last_block_with_witness = bw
-        .blocks_and_witnesses
-        .last()
-        .ok_or_else(|| anyhow::anyhow!("Unexpected test with no blocks: {}", &bw.name))?
-        .clone();
```

Comment:
This and the below previous `for` are gone since now we have `BlockAndWitness` and not `BlocksAndWitnessess`.